.p4a

```diff
@@ -33,3 +33,7 @@
 --depend com.google.firebase:firebase-crashlytics-ndk:18.3.2
 --depend com.google.firebase:firebase-analytics:21.2.0
 --with-debug-symbols
+--debug-manifest-placeholders '[analytics_enabled: "false"]'
```

Comment:
Might be worth mentioning in the commit message that this depends on https://github.com/endlessm/python-for-android/pull/18.

Reply:
Will do.